samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&mram1x {
+	/delete-node/ cpuapp_rw_partitions;
+
+	cpuapp_rw_partitions: cpuapp-rw-partitions {
+		compatible = "nordic,owned-partitions", "fixed-partitions";
+		status = "okay";
+		perm-read;
+		perm-write;
+		perm-secure;
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		dfu_partition: partition@100000 {
+			reg = <0x100000 DT_SIZE_K(908)>;
+		};
+
+		storage_partition: partition@1e3000 {
+			reg = <0x1e3000 DT_SIZE_K(20)>;
+		};
+
+		bt_fast_pair_partition: partition@1e8fb8 {
+			label = "bt_fast_pair";
+			reg = <0x1e8fb8 0x48>;
+		};
```

Comment:
I'm not sure if the addresses are correct 20kB is equal to 0x5000 so I think the `bt_fast_pair_partition` address should be set to 0x1e8000 (0x1e3000 + 0x5000) instead of 0x1e8fb8. Correct me if I'm wrong.

Reply:
@kapi-no

Reply:
Well, I positioned it at the end of the 0x1000 memory chunk to mimick the PM solution in the case of automatic builds without the specific memory layout file (pm.yaml). On the other hand, we typically position the provisioning data at the beginning of the 0x1000 memory chunk in builds where we define the memory layout file (pm.yaml).

@alstrzebonski, I don't have a preference here. We can align it according to your suggestion or we can leave this part and think about the position of the fast pair provisioning data hex in the nRF54H20 memory in the follow-up task

Reply:
@alstrzebonski, could you create a follow-up task?

Reply:
https://nordicsemi.atlassian.net/browse/NCSDK-30269

Reply:
FYI: `0x1e8fb8` is not divisible by 128 bits, which is the "erase block" for MRAM, so this assignment is OK as long as you do not need to erase this partition using `flash_*` APIs.

Reply:
@tomchy - just to make sure - you're referring here to write-block-size which is equal to 16 bytes? That means that addresses should be divisible by 0x10. Am I correct?